README.md

```diff
@@ -1,19 +1,22 @@
 # Open Swiss Buildings API
 
 This is an API for the [Swiss Buildings Register](https://www.housing-stat.ch/) (Eidgenössisches Gebäude- und Wohnungsregister GWR).
+The API dataset has been extended to also include building information taken from the
+[Liechtenstein Building Registry](https://www.statistikportal.li/de/erhebungen-register/gebaeude-und-wohnungsregister).
+
 It provides ways to resolve input data into address lists.
 Additionally, it provides an autocomplete-search for addresses.
 
 ## Background
 
-The application fetches the data about buildings from the [Swiss Buildings Register](https://www.housing-stat.ch/)
-and stores them in a PostgreSQL database.
+The application fetches the data about buildings from the two registries (for Switzerland and Liechtenstein) and stores
+them in a PostgreSQL database.
 
 For the autocomplete and address resolving, the data is also stores in a search engine.
 
 ## Usage
 
-The API is able to resolve various input data into lists of addresses containing the Swiss building ID (EGID) and its coordinates.
+The API is able to resolve various input data into lists of addresses containing the building ID (EGID or GEID) and its coordinates.
```

Comment:
did we implement GEID here? if not we should mention that for now those endpoints only search switzerland and LI is only handled in geojson.

Reply:
as I read this paragraph: it is describing the "output" of the resolving, thus it is correct to mention both EGID (for CH) and GEID (for LI)

The way we handle the "input" of the resolving is not described in the Readme at the moment.
We would need to describe the 4 of them.

Reply:
you are right. thanks for adding the new sections about supported resolving.